static/src/javascripts/projects/admin/modules/abtests/participation-item.js

```diff
@@ -23,13 +23,17 @@ class ParticipationItem extends Component {
         );
     }
 
+    config: Object;
+
     prerender(): void {
         const origin = /gutools.co.uk$/.test(document.location.origin)
             ? 'http://www.theguardian.com'
             : document.location.origin;
         const href = `${this.config.examplePath}=${this.config.variant}`;
-        this.elem.textContent = this.config.variant;
-        this.elem.href = origin + href;
+        const elem = ((this.elem: any): HTMLLinkElement);
```

Comment:
To avoid this cast, shall we make the type of `this.elem: ?HTMLElement | ?HTMLLinkElement`

?

Reply:
Yeah, we could. I'm afraid it would become a growing list, but I think you're right. 👍